app/src/main/java/com/example/vmedvediev/ua21summerdancecamp/UI/SplashActivity.kt

```diff
@@ -17,15 +17,14 @@ import java.nio.charset.Charset
 class SplashActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        setTheme(R.style.AppTheme)
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_splash)
         when {
             EventsCache.eventsList.isNotEmpty() -> {
                 startActivity(Router.prepareMainActivityIntent(this))
                 finish()
             }
-            EventsDatabaseHelper.getEvents().isNotEmpty() -> {
+            DatabaseHelper.getEvents().isNotEmpty() -> {
```

Comment:
You should get events from repository, not from db helper. Because repository incapsulates database work